hera_mc/cm_utils.py

```diff
@@ -230,6 +233,19 @@ def listify(X):
     return [X]
 
 
+def match_list(a_obj, b_obj, upper=False):
```

Comment:
Needs a docstring which should mention the trickiness you're doing below where you're replicating `b_obj` to have the same length as `a_obj` if `b_obj` isn't a list.